config/locales/en.yml

```diff
@@ -392,7 +392,7 @@ en:
       status:
         delegate_regions:
           trainee_delegate: "Trainee Delegate"
-          candidate_delegate: "Junior Delegate"
+          junior_delegate: "Junior Delegate"
```

Comment:
Just double-checking to make sure this value actually needs to be changed?
I mean, in the database it will still be `candidate_delegate`

Reply:
Resolving this as we discussed this over slack.